package.json

```diff
@@ -50,6 +72,8 @@
   "devDependencies": {
     "bower": "^1.7.1",
     "tslint": "^3.14.0",
+    "electron-builder": "^5.22.2",
+    "electron-prebuilt": "^1.3.2",
```

Comment:
This is a big issue. electron-builder asks me to move electron-prebuilt to dev dependency. Maybe we need a two different package.json?